### PR TITLE
[FTWDLISJULY22] Jason Rosa

### PR DESCRIPTION
**Pull request by JasonRosa**
Xico - I did as much as I could but my terminal stopped working, I tried making adjustments to various things and couldn't figure out. Please can we go over tomorrow? I know its important for the project.

### CODE REVIEW

File: routes/celebrities.routes.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const router = require("express").Router();
+const Celebrities = require('/models/celebrities.model')
```
Comment (webLisbonTA) [inline routes/celebrities.routes.js:2]: Here, the error on the terminal was here,
You should write like this: const Celebrity = require("../models/celebrities.model")

File: routes/movies.routes.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const router = require("express").Router();
+const Movies = require('/models/movies.model')
+const Celebrities = require('/models/celebrities.model')
```
Comment (webLisbonTA) [inline routes/movies.routes.js:3]: Here, the error on the terminal was here,
You should write like this: 
const Movie = require("../models/movies.model")
const Celebrity = require("../models/celebrities.model")

**Review by webLisbonTA — COMMENTED** [review]
Good job on the lab!

File: routes/celebrities.routes.js (hunk 1 above)
Comment (JasonRosa) [reply routes/celebrities.routes.js:2]: so it takes ..? sometimes I find I do it and it gives me issues and sometimes I don't and its fine. I will add them back in and see if it works, thanks

**Pull request closed without merging**